### ISSUE TITLE
Avatars don't show up properly

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#14]
```
Some of the avatars in the timeline do not show up correctly. Instead of
resizing them, it shows the top left hand corner of it's original uploaded
size.

I'm using Pino version 0.1.0b3 on Ubuntu 9.10.
```

Original issue reported on code.google.com by `aaronfou...@gmail.com` on 17 Dec 2009 at 3:00

**Comment by GoogleCodeExporter** [comment]
```
Can you attach some screenshot?
```

Original comment by `tro...@gmail.com` on 17 Dec 2009 at 3:06

**Comment by GoogleCodeExporter** [comment]
```
Here are some screenshots of how my timeline looks in Pino with the affected 
avatars
compared to how it looks on Twitter.
```

Original comment by `aaronfou...@gmail.com` on 17 Dec 2009 at 3:53

Attachments:
- [Screenshot-pino.png](https://storage.googleapis.com/google-code-attachments/pino-twitter/issue-14/comment-2/Screenshot-pino.png)
- [Twitter-Home_1261064767942.png](https://storage.googleapis.com/google-code-attachments/pino-twitter/issue-14/comment-2/Twitter-Home_1261064767942.png)
- [Screenshot-pino-1.png](https://storage.googleapis.com/google-code-attachments/pino-twitter/issue-14/comment-2/Screenshot-pino-1.png)
- [Twitter-Home_1261064824194.png](https://storage.googleapis.com/google-code-attachments/pino-twitter/issue-14/comment-2/Twitter-Home_1261064824194.png)

**Comment by GoogleCodeExporter** [comment]
```
Closed by f8022f4c46f8 revision.

Thanks.
```

Original comment by `tro...@gmail.com` on 17 Dec 2009 at 4:09
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
You're welcome!

And thanks for fixing it. That was very quick! :)
```

Original comment by `aaronfou...@gmail.com` on 17 Dec 2009 at 5:06
